cypress/features/transactions.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Money
+
+  As a user of the Prisoner Content Hub
+  I want to be able to check my transactions
+  So that I can manage my money
+
+  Background: I am signed in and on the money page
+    Given that I am signed in on the money page
+
+  Scenario: I want to navigate to the Transactions page
+    When I click the "view your transactions" link
+    Then I am on the "Transactions" page
+
+
+
+
```

Comment:
Did we agree to remove these and use vanilla cypress for now (applies to the money feature above as well)? 

I think our end-to-end tests need to be explicitly testing a valuable thing for the user. Because they're quite slow (relative to unit/integration tests) we'll end up with fewer of them, so each one wants to be carrying its own weight. 

It might just be a side-effect of trialling gherkin/cucumber here, but I think this could be a much richer test, for example checking that the correct transactions were shown in the correct order. This will be _much_ easier with the month selection addition that @LucasCairns demoed today – a single user journey can exercise the being on the page, the initial transaction set and the functionality of the date picker. So maybe don't worry about coming up with a great test for this PR.

Reply:
Gone! 💥